backend/src/controllers/club_poc.go

```diff
@@ -0,0 +1,180 @@
+package controllers
+
+import (
+	"github.com/GenerateNU/sac/backend/src/errors"
+	"github.com/GenerateNU/sac/backend/src/models"
+	"github.com/GenerateNU/sac/backend/src/services"
+	"github.com/gofiber/fiber/v2"
+)
+
+type ClubPointOfContactController struct {
+	clubPointOfContactService services.ClubPointOfContactServiceInterface
+}
+
+func NewClubPointOfContactController(clubPointOfContactService services.ClubPointOfContactServiceInterface) *ClubPointOfContactController {
+	return &ClubPointOfContactController{clubPointOfContactService: clubPointOfContactService}
+}
+
+// GetClubPointOfContacts godoc
+//
+// @Summary		Retrieve all point of contacts for a club
+// @Description	Retrieves all point of contacts associated with a club
+// @ID			get-point-of-contacts-by-club
+// @Tags      	club-point-of-contact
+// @Produce		json
+// @Param		clubID	path	string	true	"Club ID"
+// @Success		200	  {object}	    []models.PointOfContact
+// @Failure     400   {object}      errors.Error
+// @Failure     404   {object}      errors.Error
+// @Failure     500   {object}      errors.Error
+// @Router		/clubs/{clubID}/pocs/  [get]
+func (u *ClubPointOfContactController) GetClubPointOfContacts(c *fiber.Ctx) error {
```

Comment:
```suggestion
func (poc *ClubPointOfContactController) GetClubPointOfContacts(c *fiber.Ctx) error {
```
nit but why use `u` for the pointer receiver